class.jetpack.php

```diff
@@ -6659,38 +6656,6 @@ public static function get_jetpack_options_for_reset() {
 		return Jetpack_Options::get_options_for_reset();
 	}
 
-	/**
-	 * Check if an option of a Jetpack module has been updated.
-	 *
-	 * If any module option has been updated before Jump Start has been dismissed,
-	 * update the 'jumpstart' option so we can hide Jump Start.
-	 *
-	 * @param string $option_name
-	 *
-	 * @return bool
-	 */
-	public static function jumpstart_has_updated_module_option( $option_name = '' ) {
-		// Bail if Jump Start has already been dismissed
-		if ( 'new_connection' !== Jetpack_Options::get_option( 'jumpstart' ) ) {
-			return false;
-		}
-
-		$jetpack = Jetpack::init();
-
-		// Manual build of module options
-		$option_names = self::get_jetpack_options_for_reset();
-
-		if ( in_array( $option_name, $option_names['wp_options'] ) ) {
-			Jetpack_Options::update_option( 'jumpstart', 'jetpack_action_taken' );
```

Comment:
Seems like we could get rid of any `jetpack_action_taken` occurrences as potential Jumpstart option values in the Jetpack codebase too. Seems like the only remaining usage is around `jetpack_show_jumpstart()`.

Reply:
Thanks! Addressed feedback in 02cd67b638e049e6a5c34c9873f79cabf6ef6567. Updated the PR description to clarify what it's doing now